taxi/views.py

```diff
@@ -86,4 +87,31 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
-    queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+    queryset = Driver.objects.prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+def assign_unassign_to_car(request, *args, **kwargs):
+    car_pk = kwargs["pk"]
+    users_cars = request.user.cars
+    get_object_or_404(Car, id=car_pk)
+    if users_cars.filter(id=car_pk):
+        users_cars.remove(car_pk)
+    else:
+        users_cars.add(car_pk)
+    return redirect("taxi:car-detail", pk=car_pk)
```

Comment:
use HttpResponseRedirect with `reverse_lazy()` and provide pk in `args=`. It would be much more better